app/tutorial_servo_control/src/servo_control.h

```diff
@@ -0,0 +1,17 @@
+#ifndef TUTORIAL_SERVO_H
+#define TUTORIAL_SERVO_H
+#include "mbed.h"
+
+class TutorialServo{
+private:
+    float m_servoRangeInDegrees;
+    PwmOut mservoPwmOut(PA_1);

```

Comment:
This pin should be configurable by the constructor (the header file should be unchanged from the one given in the training document)